### PR TITLE
Fixed #24289 -- Reversed usage of Field.many_to_one and one_to_many.

### PR DESCRIPTION
**Pull request by loic**

### CODE REVIEW

**Comment by carljm** [comment]
LGTM! If Jenkins is happy, I say merge away.

File: docs/ref/models/fields.txt [hunk 1]
```diff
@@ -1810,10 +1810,16 @@ relation. These attribute are present on all fields; however, they will only
 have meaningful values if the field is a relation type
 (:attr:`Field.is_relation=True <Field.is_relation>`).
 
+.. attribute:: Field.many_to_one
```
Comment (timgraham) [inline docs/ref/models/fields.txt]: (the ordering for fields and attributes had been alphabetical)

Comment (loic) [inline docs/ref/models/fields.txt]: I figured but for documentation I think it's quite confusing to start with the exceptions "reverse fields" and something from contrib.

Here we start with the forward field for FK, followed by its reverse, that's IMO more logical, what do you think?

Comment (timgraham) [inline docs/ref/models/fields.txt]: I won't veto.

Comment (loic) [inline docs/ref/models/fields.txt]: Actually should I move many_to_many to the top? Then it'd be effectively alphabetical.

Comment (timgraham) [inline docs/ref/models/fields.txt]: hm, m does come before o ... :abcd:

**Pull request merged**